usr/lib/systemd/system/proxysql.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=High Performance Advanced Proxy for MySQL
+
+[Service]
+Type=forking
+PIDFile=/var/lib/proxysql/proxysql.pid
+ExecStart=/usr/local/bin/proxysql
```

Comment:
I'd recommend the following:
Environment="CONFIGFILE=/etc/proxysql.cnf DATADIR=/var/lib/proxysql"
EnvironmentFile=-/etc/sysconfig/proxysql   (the '-' indicates that it doesen't matter if the file is missing)
ExecStart=/usr/local/bin/proxysql -f -c ${CONFIGFILE} -d ${DATADIR}
If you leave Type=simple (default) and add the '-f' you can skip the whole pid file and systemd will manage the process directly.
With the usage of the environment vars you'll have sane defaults but can easily override them by placing the sysconfig/proxysql file if needed.
Maybe add 
LimitNOFILE=XXX
to increase file handle limits. Not sure how many proxysql might need.